fdbclient/NativeAPI.actor.cpp

```diff
@@ -2233,14 +2233,16 @@ ACTOR Future<Standalone<VectorRef<const char*>>> getAddressesForKeyActor(Key key
 	// If key >= allKeys.end, then getRange will return a kv-pair with an empty value. This will result in our serverInterfaces vector being empty, which will cause us to return an empty addresses list.
 
 	state Key ksKey = keyServersKey(key);
+	state Standalone<RangeResultRef> serverTagResult = wait( getRange(cx, ver, lastLessOrEqual(serverTagKeys.begin), firstGreaterThan(serverTagKeys.end), GetRangeLimits(CLIENT_KNOBS->TOO_MANY), false, info ) );
```

Comment:
@alexmiller-apple I just came across this change. Would it be reasonable to run these two range queries concurrently?

Reply:
It would be, and I think this line of code is going away anyway.